File: TowerReactDotNet/Models/TowerEvent.cs
namespace TowerReactDotNet.Models
{
    public class TowerEvent
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string CoverImg { get; set; }
        public string Location { get; set; }
        public int Capacity { get; set; }
        public string StartDate { get; set; }
        public string Type { get; set; }
        public bool IsCanceled { get; set; }
        public string CreatorId { get; set; }
        public Profile Creator { get; set; }
    }
}

File: TowerReactDotNet.client/src/models/TowerEvent.js
export class TowerEvent{
    constructor(data){
        this.id = data.id
        this.name = data.name
        this.description = data.description
        this.coverImg = data.coverImg
        this.location = data.location
        this.capacity = data.capacity
        this.date = new Date(data.startDate).toLocaleDateString()
        this.type = data.type
        this.isCanceled = data.isCanceled
        this.creatorId = data.creatorId
        this.creator = data.creator
    }
}